### DISCUSSION TITLE
Disable Meal Planning and Shopping List (i.e. Use for recipes only)

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1386] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
For users who only want the recipes feature, the Meal Planning and Shopping List tabs are unnecessary. It'd be nice to have the option to simplify the interface by removing these tabs via settings.

**(Optional): Suggest A Solution**
Add two new checkboxes in settings to hide Meal Planning and Shopping List tags. Potentially, also hide features relating to these tabs.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by JohanAlbrectsen** [comment]
I'd really like this!
